test/c/simulation.cpp

```diff
@@ -0,0 +1,131 @@
+#include <gtest/gtest.h>
+#include <mpi.h>
+
+extern "C" {
+    #include "../src/trixi.h"
+}
+
+// Julia project path defined via cmake
+const char * julia_project_path = JULIA_PROJECT_PATH;
+
+// Example libexlixir
+const char * libelixir_path =
+  "../../../LibTrixi.jl/examples/libelixir_p4est2d_dgsem_euler_sedov.jl";
+
+TEST(CInterfaceTest, SimulationRun) {
+
+    // Initialize MPI
+    int argc = 0;
+    char *** argv = NULL;
+    int provided_threadlevel;
+    int requested_threadlevel = MPI_THREAD_SERIALIZED;
+    MPI_Init_thread(&argc, argv, requested_threadlevel, &provided_threadlevel);
+
+    MPI_Comm comm = MPI_COMM_WORLD;
+
+    int rank;
+    MPI_Comm_rank(comm, &rank);
+
+    int nranks;
+    MPI_Comm_size(comm, &nranks);
+
+    // Initialize libtrixi
+    trixi_initialize( julia_project_path, NULL );
```

Comment:
```suggestion
    trixi_initialize(julia_project_path, NULL);
```
I think you've been somewhat inconsistent about whether to include spaces inside parens, so maybe you need to make up your mind and just use one style everywhere 😬